package.json

```diff
@@ -5,84 +5,61 @@
   "description": "Hypertrons Chromium Extension",
   "license": "Apache",
   "scripts": {
-    "start": "node utils/webserver.js",
-    "watch": "cross-env NODE_ENV=development node utils/build.js",
-    "watch:mock": "cross-env NODE_ENV=development MOCK=true node utils/build.js",
-    "web-ext": "web-ext run --target=chromium",
-    "build": "cross-env NODE_ENV=production node utils/build.js",
-    "prettier": "prettier --write '**/*.{js,jsx,css,html}'"
+    "build": "node utils/build.js",
+    "start": "node utils/server.js",
+    "prettier": "prettier --write '**/*.{js,jsx,ts,tsx,json,css,scss,md}'"
   },
   "dependencies": {
-    "@antv/g6": "^4.5.3",
-    "@antv/graphin": "^2.0.6",
-    "@antv/graphin-components": "^2.0.7",
-    "@fluentui/react": "^7.121.1",
-    "@fluentui/react-hooks": "^8.2.0",
-    "@hot-loader/react-dom": "^16.13.0",
-    "@octokit/core": "^3.4.0",
-    "@types/chrome": "0.0.104",
-    "@types/react": "^17.0.2",
-    "copy-to-clipboard": "^3.3.1",
-    "delegated-events": "^1.1.2",
+    "@antv/g6": "^4.5.5",
+    "@antv/graphin": "^2.4.14",
+    "@antv/graphin-components": "^2.4.0",
+    "@fluentui/react": "^8.55.1",
+    "@fluentui/react-hooks": "^8.3.12",
+    "@hot-loader/react-dom": "^17.0.2",
+    "@octokit/core": "^3.5.1",
+    "@uifabric/icons": "^7.6.2",
     "anichart": "2.4.4",
-    "echarts": "^5.0.2",
-    "github-url-detection": "^4.10.0",
+    "buffer": "^6.0.3",
+    "delegated-events": "^1.1.2",
+    "echarts": "^5.3.0",
+    "github-url-detection": "^5.8.0",
     "jquery": "^3.6.0",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-hot-loader": "^4.12.20",
-    "react-test-renderer": "^17.0.1"
+    "office-ui-fabric-react": "^7.182.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-hot-loader": "^4.13.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.9.0",
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/preset-env": "^7.9.0",
-    "@babel/preset-react": "^7.9.4",
-    "@types/jest": "^26.0.20",
-    "@types/jquery": "^3.5.5",
-    "@types/react-dom": "^17.0.1",
-    "@types/react-test-renderer": "^17.0.1",
-    "@typescript-eslint/eslint-plugin": "^4.16.1",
-    "@typescript-eslint/parser": "^4.16.1",
-    "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.1.0",
-    "babel-preset-react-app": "^9.1.2",
-    "clean-webpack-plugin": "^3.0.0",
-    "copy-webpack-plugin": "^5.1.1",
-    "cross-env": "^7.0.3",
-    "crx-webpack-plugin": "^0.1.6",
-    "css-loader": "^3.4.2",
-    "eslint": "^6.8.0",
-    "eslint-config-react-app": "^5.2.1",
-    "eslint-plugin-flowtype": "^4.7.0",
-    "eslint-plugin-import": "^2.20.1",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-react-hooks": "^3.0.0",
```

Comment:
eslint is not directly called by any npm scripts. Though it can be used by editor linter extensions like VS Code ESLint extension or coc-eslint. But I found pre-developers may not use these extensions because otherwise they would not ignore so many eslint errors shown in the editor.

I would like to remove eslint for now, and add it back when we prepared to make use of this tool.